tools/vendor_hid_test.py

```diff
@@ -0,0 +1,259 @@
+"""These tests verify the functionality of the VendorHID interface."""
+import hid
+import time
+import unittest
+
+_OPENSK_VID = 0x1915
+_OPENSK_PID = 0x521F
+_FIDO_USAGE_PAGE = 0xF1D0
+_VENDOR_USAGE_PAGE = 0xFF00
+_PACKETS = 4
+_PACKET_SIZE = 64
+_SEND_DATA_SIZE = _PACKET_SIZE + 1
+_DEFAULT_CID = bytes([0xFF, 0xFF, 0xFF, 0xFF])
+
+
+def sleep():
+  #time.sleep(.01)
+  pass
+
+
+def ping_data_size(packets):
+  return 57 + 59 * (packets - 1)
+
+
+_BYTE_VAL = 0x10
+
+
+def get_byte():
+  global _BYTE_VAL
+  _BYTE_VAL = _BYTE_VAL + 1
+  return _BYTE_VAL
```

Comment:
Since `_BYTE_VAL` is technically local to the function, we could also do this:
```
def get_byte():
  get_byte.byte_val += 1
  return get_byte.byte_val
get_byte.byte_val = 0x10
```

I wonder if that's worth the trouble. Sending a const per endpoint would also work for me.

Reply:
Update to use your suggested code.

I still see value in having this code. Using unique values per test (without having to manage this) allows for checking that the returned data is as expected.

Reply:
Okay sounds good!